### ISSUE TITLE
Error Spam with Core Protect Extension

### ISSUE THREAD

**Issue by mercurialmusic** [plan-player-analytics/Plan#1046]
### Describe the bug
Getting some consistent and considerable error spam with PLAN and CoreProtect. Possibly due to older version of CoreProtect? I'm not on 1.14 yet and their latest versions are 1.14 only.

### Server information
Running CraftBukkit version git-Spigot-f09662d-3697519 (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)
Plan 4.8.4
CoreProtect 2.16.3

### Exceptions & Other Logs
https://gist.github.com/mercurialmusic/78005f9bf5e879391d2f4ac1ee622c57

### Additional information
**Frequency of occurrence**: <>
Seemingly random, between 2-5 minutes apart.

**Comment by Rsl1122** [comment]
This seems like an issue in your CoreProtect database.

Plan doesn't do database calls directly (Plan uses the public API) so this is an issue in CoreProtect itself (possibly due to the older version, maybe another version has this bug fixed)

If you can't find a version of CoreProtect with the issue fixed I suggest disabling the integration in Plan config until you are able to obtain a working copy.

TLDR: Bug in CoreProtect code / database
